source/index.html

```diff
@@ -170,6 +170,15 @@
 			color: #fff;
 			background: #24CDE8;
 		}
+		
+	           h1 {
+			font-weight: 300;
+			font-size: 40px;
+			line-height: 78px;
+			text-transform: uppercase;
+			color: #404040;
+                         }  
```

Comment:
I generally try to avoid doing page-wide style changes like this.  In this case, there is a single element on the page that you want to style this way, so we should probably apply the style locally instead of globally.  This means a style like `#addonsbanner > .addonsbanner > h1 { ... }` in the `<style>` block for the section that has this element.  This way, only that element is targeted by this style rather than every `h1` element on the page.